std/traits.d

```diff
@@ -8590,3 +8591,119 @@ enum isCopyable(S) = is(typeof(
     static assert(isCopyable!int);
     static assert(isCopyable!(int[]));
 }
+
+/++
+ + Determines if `F` is a strongly `pure` function. To be strongly `pure`, a function must satisfy the following:
+ + $(UL
+ +   $(LI Each parameter is `immutable`, OR can be converted automatically to `immutable`
```

Comment:
Lower case OR and AND, maybe. Readers are assumed to know a bit boolean logic, no ?

Reply:
Done